lib/config.js

```diff
@@ -25,6 +25,13 @@ export let config = {
         type: 'boolean',
         default: false,
     },
+    newFileConfirmation: {
+        title: 'Prompt confirmation on new file',
```

Comment:
Maybe we should use the slightly-more-grammatical "Prompt before creating new files"?